src/index.js

```diff
@@ -14,4 +15,6 @@ ReactDOM.render(
   document.getElementById("root")
 );
 
+window.fetchContest = fetchContest
```

Comment:
is this for testing purposes? otherwise we don't need this on the window in the production version, I think.